testsuites/syncgateway/system/conftest.py

```diff
@@ -180,6 +182,7 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
 
     # Verify all sync_gateways and sg_accels are reachable
     c = cluster.Cluster(cluster_config)
+    remote_executor = RemoteExecutor(c.sync_gateways[0].ip)
```

Comment:
I don't think we can rely on this being a single node. Also. RemoteExecutor will not work with Docker. Ansible will be able to.